bench-tracer/run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+targets="k8s"
+#targets="rhel"
+#targets="k8s rhel"
+
+endpoint_arg=""
+
+csid=1
+
+for target in ${targets}; do
+    case "${target}" in
+	"k8s")
+	    remote_host=xxx.yyy.zzz
+	    endpoint_arg+=" --endpoint k8s,host:${remote_host},user:root,client:${csid},cpu-partitioning:client-${csid}:1,volumeMounts:client-1:`/bin/pwd`/volumeMounts.json,volumes:client-1:`/bin/pwd`/volumes.json,resources:client-${csid}:`/bin/pwd`/resource.json,nodeSelector:client-${csid}:`/bin/pwd`/nodeSelector.json,securityContext:client-${csid}:`/bin/pwd`/securityContext.json,masters-tool-collect:0"
+	    ;;
+	"rhel")
+	    remote_host=xxx.yyy.zzz
+	    endpoint_arg+=" --endpoint remotehost,host:${remote_host},user:root,client:${csid},cpu-partitioning:client-1:1"
```

Comment:
Minor nit here, if you are going to use the `csid` variable then all instances of `client-` or `server-` (depending on workload) need to use `client-${csid}` or `server-${csid}` -- so you should not have `client-1` anywhere.